src/providers/spacex-api/spacex-api.ts

```diff
@@ -51,4 +50,17 @@ export class SpacexApiProvider {
     const httpParams = Object.getOwnPropertyNames(params).reduce((p, key) => p.set(key, params[key]), new HttpParams());
     return this.http.get<IRocket[]>(endpointUrl, {params: httpParams});
   }
+
+  getRockets(params: any): Observable<IRocket> {
```

Comment:
The "getRockets" function already exists : "getAllRockets", they both use the same endpoint so we might not need the 2 of them.